src/IBM.Watson.VisualRecognition.v4/IBM.Watson.VisualRecognition.v4.csproj

```diff
@@ -24,11 +24,100 @@
         <DebugSymbols>true</DebugSymbols>
     </PropertyGroup>
 
+    <ItemGroup>
+      <Compile Remove="Examples\**" />
+      <EmbeddedResource Remove="Examples\**" />
+      <None Remove="Examples\**" />
+    </ItemGroup>
+
     <ItemGroup>
         <PackageReference Include="IBM.Cloud.SDK.Core" Version="1.1.0" />
-        <PackageReference Include="IBM.Watson.Common" Version="4.6.0" />
-        <PackageReference Include="JsonSubTypes" Version="1.6.0" />
+        <PackageReference Include="IBM.Watson.Common" Version="5.0.0" />
         <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+        <PackageReference Include="JsonSubTypes" Version="1.6.0" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Compile Remove="Test\Unit\VisualRecognitionUnitTests.cs" />
+      <Compile Remove="Test\Unit\VisualRecognitionServiceGeneratedUnitTests.cs" />
+      <Compile Remove="Test\Unit\obj\Debug\netcoreapp2.0\IBM.Watson.VisualRecognition.v4.UnitTests.Program.cs" />
+      <Compile Remove="Test\Unit\obj\Debug\netcoreapp2.0\.NETCoreApp,Version=v2.0.AssemblyAttributes.cs" />
+      <Compile Remove="Test\Unit\obj\Debug\netcoreapp2.0\IBM.Watson.VisualRecognition.v4.UnitTests.AssemblyInfo.cs" />
+      <Compile Remove="Test\Integration\VisualRecognitionServiceIntegrationTests.cs" />
+      <Compile Remove="Test\Integration\obj\Debug\netcoreapp2.0\IBM.Watson.VisualRecognition.v4.IntegrationTests.AssemblyInfo.cs" />
+      <Compile Remove="Test\Integration\obj\Debug\netcoreapp2.0\.NETCoreApp,Version=v2.0.AssemblyAttributes.cs" />
+      <Compile Remove="Test\Integration\obj\Debug\netcoreapp2.0\IBM.Watson.VisualRecognition.v4.IntegrationTests.Program.cs" />
+      <Compile Remove="Test\Integration\Properties\AssemblyInfo.cs" />
+    </ItemGroup>
+    <ItemGroup>

```

Comment:
I think we can remove this itemgroup if we add tests above

Reply:
Fixed!